src/toil_scripts/lib/files.py

```diff
@@ -49,6 +49,26 @@ def copy_file_job(job, name, file_id, output_dir):
     copy_files([fpath], output_dir)
 
 
+def move_files(file_paths, output_dir):
```

Comment:
This is a semi-dangerous function to use now that we have caching. Might want to put a warning in the docstring?